ui/README.md

```diff
@@ -1,34 +1,92 @@
+LOG_LEVEL=info
+```
+
+[!WARNING]
+Please generate a valid and secure value for `NEXTAUTH_SECRET`. We suggest running `openssl rand -base64 32` to get started.
+
+Install the required dependencies.
+
+```bash
+npm run install
+```
+
+Then run the application.
+
+```bash
+npm run build
+npm run start
+```
+
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the console.
+
+## Build
+
+```bash
+npm run build
+```
+
+This will create an optimized version of the application that can be deployed.
+
+## Developing the UI
 
 ```bash
 npm run dev
-# or
-yarn dev
-# or
-pnpm dev
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+This will start the UI in dev mode, hosted (by default) on port 3000. When running successfully, you should see output similar to:
+
+```
+strimzi-ui@0.1.0 dev
+NODE_OPTIONS='-r next-logger' next dev | pino-pretty
 
-You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
+[16:11:06.206] INFO (console/14593):    ▲ Next.js 14.1.4
+[16:11:06.206] INFO (console/14593):    - Local:        http://localhost:3000
+[16:11:06.206] INFO (console/14593):    - Environments: .env.local
+[16:11:06.206] INFO (next.js/14593):
+prefix: "info"
+[16:11:08.981] INFO (next.js/14593): Ready in 2.9s
+prefix: "event"
+```
 
-This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
+You can then access the UI on port 3000 or your localhost.
 
-## Learn More
+Note: you will need the REST API running for the UI to work. See the README at the root of this repository for examples of how to do that.
 
-To learn more about Next.js, take a look at the following resources:
+### Develop the components in isolation using Storybook
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+```bash
+npm run storybook
+```
 
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+This will start Storybook, hosted on port 6006.
 
-## Deploy on Vercel
+### Test the UI
 
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+```bash
+npm run build-storybook
+http-serve -p 6006 storybook-static
+
+# in a different terminal
+npm run test-storybook
+```
+
+This will build Storybook and run all the relative unit tests.
+
+```bash
+npm run build
+npm run test
+```
 
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+This will run Playwright against a production build of the application. 
```

Comment:
```suggestion
This will run E2E tests(powered by [Playwright](https://playwright.dev/)) against a production build of the application. 
```

> a production build of the application

how can I get one?